server/src/main.rs

```diff
@@ -44,13 +166,48 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await;
 
     //start server
-    println!("Starting gRPC server at: {}", full_grpc_addr);
-    Server::builder()
+    info!("Starting gRPC server at: {}", full_grpc_addr);
+    let _ = Server::builder()
         .add_service(health_service)
         .add_service(TemplateRustRpcServer::new(imp))
         .serve(full_grpc_addr)
-        .await?;
-    println!("gRPC server running at: {}", full_grpc_addr);
+        .await;
+}
+
+///Main entry point: starts gRPC Server on specified address and port
+#[tokio::main]
+#[cfg(not(tarpaulin_include))]
+
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    dotenv().ok();
+
+    {
+        let log_cfg: &str = "log4rs.yaml";
+        if let Err(e) = log4rs::init_file(log_cfg, Default::default()) {
+            println!("(logger) could not parse {}. {}", log_cfg, e);
+            panic!();
+        }
+    }
+
+    // --------------------------------------------------
+    // START REST SECTION
+    // This section should be removed if there is no REST interface
+    // --------------------------------------------------
+
+    // Allow option to only generate the spec file to a given location
+    let args = Cli::parse();
+    if let Some(target) = args.openapi {
+        return generate_openapi_spec(&target);
+    }
```

Comment:
Maybe add some instruction in README on how to use the CLI? Especially for generating openapi locally.